executor/executor_write.go

```diff
@@ -682,6 +686,9 @@ func (e *InsertValues) getRowsSelect(cols []*table.Column) ([][]types.Datum, err
 		}
 		rows = append(rows, row)
 	}
+	if e.needRebase {
```

Comment:
`getRows` also calls `fillRowData`. So I think that it only puts this judgement into the `getRowsSelect` function isn't enough.